src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -833,6 +835,7 @@ static class CurrentProcessingFileInformation {
         private long lastModifiedTime;
         @JsonProperty(PERSISTED_CURRENT_PROCESSING_FILE_HASH)
         private String fileHash;
+        private static final Logger logger = LogManager.getLogger(LogManagerService.class);
```

Comment:
do not create this. A logger already exists. `logger`

Reply:
The class is static, so it cannot use the existing logger.

Reply:
If we want to use a logger, use the right logger, which comes from the log manager service class. In order to use it, make this class non-static, then fix the issues that that makes (looks like only a test is impacted).

Reply:
I just tried to make the class non-static, it raise another issue "@Builder is not supported on non-static nested classes", and this builder is used in several places. 
Can we use the logger as @Nelsonochoam  suggested?

Reply:
yes